src/current/_includes/common/hybrid-multi-cloud.md

```diff
@@ -0,0 +1,66 @@
+CockroachDB makes no distinction between a deployment in a single cloud provider, a hybrid deployment, and a multi-cloud deployment.
+{{site.data.alerts.end}}
+
+This page provides information to help you decide if a hybrid or multi-cloud deployment is appropriate for your workload, and to optimize the performance of this type of deployment.
+
+{{site.data.alerts.callout_info}}
+CockroachDB {{ site.data.products.cloud }} does not support hybrid or multi-cloud deployments.
+{{site.data.alerts.end}}
+
+## Configure a hybrid or multi-cloud deployment
+
+To set up a hybrid or multi-cloud deployment:
+
+- **Each node must be able to communicate with and route traffic to each of the cluster's other nodes** using the hostnames or IP addresses referenced by the `--join` flag of the [`cockroach start`]({% link {{ page.version.version }}/cockroach-start.md %}) command. A node's network is configured at the level of the node's operating system. Nodes could be connected by peering cloud provider virtual private cloud (VPC) networks together, using a VPN, encrypted network tunnels, by a combination of routing and port forwarding, or using another mechanism.
+
+    CockroachDB could route SQL client requests to any cluster node, regardless of where the client connection originates.
+- **There must be no overlapping IP address ranges** across the cluster. It is an error for multiple nodes to have the same IP address or to resolve to the same hostname.
+
+    DNS name resolution is particularly complex in a Kubernetes deployment. We recommend that you replace `kube-dns` with Core DNS. Refer to [Deploy CockroachDB on GKE](https://github.com/mbookham7/crdb-multi-cloud-k8s/blob/master/markdown/5-deploy-cockroach.md) for details.
+- **In a multi-region deployment, we recommend that you use manifests**. Refer to [Deploy multi-cloud CockroachDB on GKE](https://github.com/mbookham7/crdb-multi-cloud-k8s/blob/master/markdown/5-deploy-cockroach.md) for details.
+- **Each node deployed in the same environment must share locality** to ensure that the cluster's replicas are spread across deployment environments and to prevent single points of failure or hot spots. To specify a node's locality, pass the `--locality` flag when [starting a node]({% link {{ page.version.version }}/cockroach-start.md %}). This flag accepts an arbitrary set of key-value pairs that describe the location of the node. For example, if nodes are deployed in both Azure and Digital Ocean, you could set each node's locality to either `--locality data-center=azure` or `--locality data-center=digital-ocean`.
+
+    If nodes are deployed in multiple regions within the same cloud provider, specify the region as an additional locality, to ensure that replicas are spread across regions in each cloud provider's infrastructure.
+
+## Simulate an outage
+
+This section shows some ways to simulate an outage and validate disaster recovery procedures in a hybrid or multi-cloud deployment.
```

Comment:
Can we add a sentence along the lines of "you should run this testing to ensure you've correctly set up the locality flags"?

Reply:
Done